### DISCUSSION TITLE
daterangepicker::before not compatible with dark mode

### DISCUSSION THREAD

**Discussion by devhoussam1998** [malzariey/filament-daterangepicker-filter#51] [Q&A]
<img width="653" alt="Screenshot 2023-11-08 at 18 37 39" src="https://github.com/malzariey/filament-daterangepicker-filter/assets/121255432/0db6c0dd-0e86-4fc6-ad2c-574e34f5dbfb">

**Comment by malzariey** [comment]
What exactly is the problem?

**Reply by devhoussam1998** [reply]
check the caret it's white in I'm in dark mode
